apps/web/src/pages/Classifications/ViewClassificationPage.tsx

```diff
@@ -0,0 +1,224 @@
+import { useIntl } from "react-intl";
+import { useQuery } from "urql";
+import IdentificationIcon from "@heroicons/react/24/outline/IdentificationIcon";
+
+import {
+  Pending,
+  NotFound,
+  CardBasic,
+  Heading,
+  Separator,
+  Link,
+} from "@gc-digital-talent/ui";
+import { commonMessages } from "@gc-digital-talent/i18n";
+import {
+  FragmentType,
+  getFragment,
+  graphql,
+  Scalars,
+} from "@gc-digital-talent/graphql";
+import { ROLE_NAME } from "@gc-digital-talent/auth";
+
+import SEO from "~/components/SEO/SEO";
+import useRoutes from "~/hooks/useRoutes";
+import useRequiredParams from "~/hooks/useRequiredParams";
+import Hero from "~/components/Hero";
+import useBreadcrumbs from "~/hooks/useBreadcrumbs";
+import RequireAuth from "~/components/RequireAuth/RequireAuth";
+import pageTitles from "~/messages/pageTitles";
+import { getClassificationName } from "~/utils/poolUtils";
+import adminMessages from "~/messages/adminMessages";
+import FieldDisplay from "~/components/ToggleForm/FieldDisplay";
+
+import messages from "./messages";
+
+const ViewClassification_Fragment = graphql(/* GraphQL */ `
+  fragment ViewClassification on Classification {
+    id
+    name {
+      en
+      fr
+    }
+    group
+    level
+    minSalary
+    maxSalary
+  }
+`);
+
+interface ViewClassificationProps {
+  query: FragmentType<typeof ViewClassification_Fragment>;
+}
+
+const ViewClassification = ({ query }: ViewClassificationProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+  const classification = getFragment(ViewClassification_Fragment, query);
+
+  const pageTitle = getClassificationName(classification, intl);
+  const subTitle = intl.formatMessage(messages.classificationInfo);
+
+  const navigationCrumbs = useBreadcrumbs({
+    crumbs: [
+      {
+        label: intl.formatMessage(pageTitles.classifications),
+        url: paths.classificationTable(),
+      },
+      {
+        label: pageTitle,
+        url: paths.classificationView(classification.id),
+      },
+    ],
+  });
```

Comment:
Crumbs vary

Current

![image](https://github.com/user-attachments/assets/328aa79f-6503-48aa-8e56-16a9faa6c9f3)

Figma (ignoring the admin dashboard one which is the same as the home link)

![image](https://github.com/user-attachments/assets/6a8b5cd5-b665-4562-9fd8-f193b4fde9ce)